packages/react/src/components/actions/tests/Action.spec.tsx

```diff
@@ -16,91 +16,44 @@ describe('Actions', () => {
         };
     });
 
-    it('should render without errors', () => {
-        expect(() => {
-            shallow(<Action action={action} />);
-        }).not.toThrow();
-    });
-
     it('should have a defaultProp hideDisabled set to true', () => {
         expect(Action.defaultProps.hideDisabled).toBe(true);
     });
 
-    it('should add the id of the action on a data-trigger attribute', () => {
-        const expectedId = 'cut';
-        const component = shallow(<Action action={{...action, id: expectedId}} />);
-
-        expect(component.find('.action-label').prop('data-trigger')).toBe(expectedId);
-    });
-
-    it('should add the iconClass of the action on the icon class', () => {
+    it('should add the iconClass of the action on the icon class', async () => {
         const iconClass = 'bloup';
-        const component = shallow(<Action action={{...action, icon: 'link', iconClass}} />);
+        render(<Action action={{...action, icon: DeleteSize24Px, iconClass}} />);

```

Comment:
instead of spreading the action I would define it in every test. It's easier to understand even if we repeat a bit of code